rhizome/host/lib/vm_setup.rb

```diff
@@ -649,7 +650,7 @@ def prepare_pci_devices(pci_devices)
   end
 
   def install_systemd_unit(max_vcpus, cpu_topology, mem_gib, storage_params, nics, pci_devices, slice_name, cpu_percent_limit)
-    cpu_setting = "boot=#{max_vcpus},topology=#{cpu_topology}"
+    fail "BUG" if /["'\s]/.match?(cpu_topology)
```

Comment:
Previous `fail` would also catch issues with `max_vcpus` in addition to `cpu_topology`. Is that not an issue?